kitsune/settings.py

```diff
@@ -1252,6 +1252,7 @@ def filter_exceptions(event, hint):
 # GraphQL configuration
 GRAPHENE = {
     "SCHEMA": "kitsune.schema.schema",
+    "MIDDLEWARE": ("kitsune.graphql.middleware.DisableIntrospectionMiddleware",),
```

Comment:
Maybe we can add the middleware only when DEV is False?

Reply:
There's also this in the graphene documention
https://docs.graphene-python.org/en/latest/execution/queryvalidation/#disable-introspection

Reply:
I saw that too, but couldn't make sense of how to use the validator, because the `validate` call appears to be something that has to be embedded in the query processing flow.

Reply:
@akatsoulas For ease of testing, I decided to always add `DisableIntrospectionMiddleware`, and dynamically check for `settings.DEV` in the middleware itself. If you'd still prefer that `DisableIntrospectionMiddleware` be added conditionally, I'll do that and remove the `DEV=True` part of the tests.

Reply:
@akatsoulas After digging into the source code of the `graphene` and `graphene-django` packages, I'm convinced that the middleware approach, rather than the validation rule, is by far the simplest way to achieve the same goal of disabling introspection.